### PR TITLE
primer - russian translation

### PR DESCRIPTION
**Pull request by lgb**

### CODE REVIEW

File: primer/primer.ru.robomarkup [hunk 1]
```diff
@@ -364,24 +364,24 @@ fun sum(list[])
 seq nums[] = 1, 2, 3
 var total = sum(nums[])
 
-total is 6
+# total будет равен 6
```
Comment (alexheretic) [inline primer/primer.ru.robomarkup]: This should be left as is, I think. `total is 6` isn't a comment, when this example runs the extra line allows the inline debug for `var total = sum(nums[])` to show up better.

**Comment by alexheretic** [comment]
Other than my 1 minor comment, this looks brilliant thanks!

**Comment by lgb** [comment]
> Other than my 1 minor comment, this looks brilliant thanks!

I changed this line back to original

**Comment by alexheretic** [comment]
Nice one! That's the first primer translation in.

What do you think about the function docs translation? That would be great to see if you get time. I think it may be easier to translate that compared to the primer.

**Comment by lgb** [comment]
I think it is possible, but I just don't want to translate more than one doc in one pull-request.

**Comment by alexheretic** [comment]
Of course, I'm just letting you know it's there in case you get some more time later! :)

**Comment by alexheretic** [comment]
I've made a temporary branch on steam investigating something else, but it also has the latest translation on it. So you can see the primer translation on the `test` branch if you fancy.

**Pull request merged**